docs/docs/forms.md

```diff
@@ -76,6 +76,8 @@ handleChange(event) {
 }
 ```
 
+However, if `this.state.value` becomes `null` or `undefined` at any time (including the initial state), React will evaluate the component as uncontrolled and may produce a warning: `Input elements should not switch from uncontrolled to controlled (or vice versa)`.
+
```

Comment:
What about users who click on the link in the actual warning? I thought it might be good to provide something relevant to that warning. For me, I knew exactly how to fix, but before I got the connection from the warning to the `!= null` condition, I had literally no idea what the *** happened why I got the stupid warning...

@aweary is your experience with this warning different? I am not trying to solve any other use cases right now, just a life line for anyone who clicks the link from the warning...